scripts/install.ps1

```
@@ -0,0 +1,122 @@
+      "Downloading and verifying Dojang $version into $downloadDirectory."
+  }
+  Invoke-WebRequest `
+    -Uri "$baseUrl/$version/$asset" `
+    -OutFile $archive `
+    -UseBasicParsing
+  Invoke-WebRequest `
+    -Uri "$baseUrl/$version/SHA256SUMS" `
+    -OutFile $checksums `
+    -UseBasicParsing
+
+  $matches = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)